apidoc/Polyline.yml

```diff
@@ -31,6 +31,11 @@ properties:
     summary: Line width in pixels to use when drawing the polyline.
     type: Number
     default: 10
+    
+  - name: touchEnabled
+    summary: Determines whether view should receive touch events.
+    type: Boolean
+    default: true
```

Comment:
Actually iOS can solve this very easy, also for polygon and circle as well. If we do this, we should do it for both platforms and all overlay-types. Can you do that on Android? Then I'll add my iOS changes here.